packages/e2e-cypress/src/install.js

```diff
@@ -105,6 +105,8 @@ module.exports = async function (api) {
     devServerPort,
     shouldAddCodeCoverage,
     shouldSupportTypeScriptAndVite: shouldSupportTypeScript && api.hasVite,
+    // See https://github.com/quasarframework/quasar-testing/issues/379
+    requiresPublicPath: api.hasVite && api.hasPackage('@quasar/app-vite', '^2.0.0-0')
```

Comment:
```suggestion
    requiresPublicPath: api.hasVite && api.hasPackage('@quasar/app-vite', '^2.0.0')
```

Reply:
The aim here was to support  `@quasar/app-vite` pre-release versions. Maybe `>=2.0.0-0 <3.0.0` should be better OR don't we need to? In which case we need to upgrade the `test-vite-2` app. It's currently using a pre-release version. I plan to update it in a different PR.

Reply:
We shouldn't specifically support pre-release versions. But, test-vite-2 being old is a valid concern to do this. We need to upgrade that as well, so it's good that you are planning to do it.

After seeing this and giving it a bit of thought, I am updating my suggestion:
```suggestion
    requiresPublicPath: api.hasVite && api.hasPackage('vite', '>=5')
```
We are actually not interested in app-vite v2, we are actually interested in a change that happened in Vite 5 and still exists. So, doing this would work even if someone managed to get app-vite v1 with Vite 5 by overriding, applying patches, etc.

Reply:
Sounds good.